src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
I commented on an old version and I think it got dropped, but I was suggesting the same as you have here, adding other player-level methods so we're not overloading what the playlist property is doing.